sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerAsyncClient.java

```diff
@@ -158,6 +159,14 @@ public String getAccountName() {
         return this.accountName;
     }
 
+    /**
+     * Gets the service version the client is using.
+     *
+     * @return the service version the client is using.
+     */
+    public String getServiceVersion() {
```

Comment:
Do we need to expose this as a public rather than package-private variable? 🤔 The reason is that we'll have to pass this public API change through the board.

Reply:
Good question, we've generally erred on having the build of the client being more public in storage to allow jumping through the hierarchy more easily. The tough part is at some point or another this needs to surface through public API to be accessible in the specialized/root package divide.

@JonathanGiles mind chiming in on this?

Reply:
I believe this has changed more recently to return BlobServiceVersion, however I would rather it not be public API. There are ways to enable this (with *Helper classes), so I would encourage some degree of investigation so that, ideally, our storage APIs are only public when the target user is the end-developer, and not our code (which indicates a failing on our part).